### PR TITLE
fixed $source null check, reference to $Params

### PR DESCRIPTION
**Pull request by dan-petty**
https://github.com/ansible/ansible-modules-core/issues/3746

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @mixmatch. To the current maintainers, @trondhindenes please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

File: windows/win_feature.ps1 [hunk 1]
```diff
@@ -63,7 +63,7 @@ If ($state -eq "present") {
         $InstallParams.add("IncludeManagementTools",$includemanagementtools)
     }
     
-    if ($source -ne $null)
```
Comment (trondhindenes) [inline windows/win_feature.ps1:66]: Maybe even 
`(($source -ne $null) -and ($source -ne ""))`

**Comment by trondhindenes** [comment]
looks good @mixmatch, I added a comment in the code, would you mind adding that to the pr before merging? Might as well get it as solid as possible.

**Comment by dan-petty** [comment]
Two minutes of internet research indicates that this is unnecessary:
`(($source -ne $null) -and ($source -ne ""))`

Other [portions](https://github.com/mixmatch/ansible-modules-core/blob/a4e6e6e2ef0b3b7e3314ac9d60a6f3f2bac93f1b/windows/win_feature.ps1#L46) of the code check for variable definition using only:
`($variablename)`

There is really no acceptable value for $source that would evaluate to false, meaning this suggestion unnecessarily complicates the code and introduces bugs like the one this PR is aiming to fix.
https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=powershell%20null
https://www.codykonior.com/2013/10/17/checking-for-null-in-powershell/
https://noteits.net/2015/11/05/powershell-check-variable-for-null/
http://www.thomasmaurer.ch/2010/07/powershell-check-variable-for-null/
http://bartwullems.blogspot.com/2010/06/testing-null-values-with-powershell.html
https://www.reddit.com/r/PowerShell/comments/3i8ex6/how_to_deal_with_null_or_empty_values/
http://techibee.com/powershell/check-if-a-string-is-null-or-empty-using-powershell/1889
http://stackoverflow.com/questions/13738634/how-can-i-check-if-a-string-is-null-or-empty-in-powershell

**Comment by trondhindenes** [comment]
I'm sorry. I commented earlier today while distracted with something else.
PR looks good, shipit

**Comment by gregdek** [comment]
Thanks again to @mixmatch for this PR, and thanks @trondhindenes for reviewing. Marking for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by nitzmahone** [comment]
cherry-picked to stable-2.1

**Pull request merged**